buildform.js

```diff
@@ -20,8 +20,14 @@ var focusnicknames = "";
 var parentmarriageid = "";
 var geounique = [];
 var geocleanup = [];
+//ajout QP
+var husbandlastname = "";
+var wifelastname = "";
+const datelimit = 1000;
```

Comment:
1600 was the agreed upon limit by the curators to prevent duplication in the world tree.  If there is a new consensus, I'm fine to change it, but not until then.